File: src/eu/iamgio/chorus/variable/Variables.java
package eu.iamgio.chorus.variable;

import eu.iamgio.chorus.menus.variables.VariablesMenu;

import java.util.List;

/**
 * @author Gio
 */
public final class Variables {

    private Variables() {}

    public static List<Variable> getVariables() {
        return VariablesMenu.getInstance().getTable().getItems();
    }
}

File: src/eu/iamgio/chorus/menubar/edit/Variables.kt
package eu.iamgio.chorus.menubar.edit

import eu.iamgio.chorus.menubar.MenuBarAction
import eu.iamgio.chorus.menus.variables.VariablesMenu

/**
 * @author Gio
 */
class Variables : MenuBarAction {

    override fun onAction() {
        VariablesMenu.getInstance().show()
    }
}